sa/migrations.sh

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env bash
```

Comment:
There seems to be a split of use of brace expansion $foo vs ${foo}, and it doesn't look like in all cases they are strictly needed. I think it is a good practice to be explicit and would encourage it on all variables.

Reply:
I saw this was already brought up. I'll close this comment.